### DISCUSSION TITLE
SRBench Competition: Updates and Plans

### DISCUSSION THREAD

**Discussion by lacava** [cavalab/srbench#131] [Announcements]
Dear all,

Huge thanks to everyone for participating in our competition this year. It was a great success, with 13 methods submitted and tested. Besides having many methods, I enjoyed having such a diverse variety. Below is some info on how and when you can expect to see results. 

# mini-workshop

Since there is a lot to cover, and we would like to have a longer discussion, we are planning to host a **remote meeting for all participants**. Teams will receive an email with a doodle poll to indicate their availability. The agenda:
 
- overview of the datasets and analysis
- invited presentations by the winning teams of each track
- discussion of manuscript plans and feedback on how to improve the competition for next time

# dissemination schedule

If you are attending [GECCO 2022](https://gecco-2022.sigevo.org/HomePage), you have already seen a presentation of our datasets and analysis, with de-identified results. And, you have seen presentations from several participants, including the synthetic track winner, at the [Symbolic Regression Workshop](https://heal.heuristiclab.com/research/symbolic-regression-workshop). 
**The official announcement of winners will occuring during the closing ceremonies tomorrow, Wednesday, 12:00 - 13:30 Eastern Time.**

For those that aren't attending, we will post the winners here shortly after, and also share a de-identified version of our presentation. Following that, the full results files and code will be uploaded to zenodo, and participants are encouraged to scan the results and post any questions or discussion. We are hoping to have the full release before the end of the week. 

Thanks again for participating. It is very exciting to see so many promising and competitive methods.

**Comment by lacava** [comment]
The results of the competition are now available at https://cavalab.org/srbench/competition-2022/. 

# Winners


Synthetic Track:
1st  - QLattice
2nd - pysr
3rd – uDSR

Real-World Track:
1st  - uDSR
2nd - QLattice
3rd - geneticengine

**GECCO Presentation (with winners and additional info):**
https://docs.google.com/presentation/d/1wKz5_nAhjLc8WHdyhOwsPjB96G4vT-4ZNJlV9QnT-Hk/edit?usp=sharing

We will continue to update the repo with results. Currently the results and post-processing are available via zenodo: https://zenodo.org/badge/DOI/10.5281/zenodo.6842176.svg

Regards,

@cavalab/srbench-comp

**Comment by janoPig** [comment]
All *featureselection*.csv data uploaded to zenodo was computed with function
 '0.11*x1^3 + 0.91*x2*x3 + 0.68*x4*x5 + 0.26*x6^2*x7 + 0.13*x8*x9*x10'
not this one
'0.11*x1^3 + 0.91*x3*x5 + 0.68*x7*x9 + 0.26*x11^2*x13 + 0.13*x15*x17*x19'
from presentation or create_data.py

Its look like feature_absence_score was wrong computed.

**Reply by lacava** [reply]
see https://github.com/cavalab/srbench/issues/132
